src/server/game/Battlegrounds/BattlegroundQueue.h

```diff
@@ -51,6 +51,7 @@ struct GroupQueueInfo                                       // stores informatio
     uint32  IsInvitedToBGInstanceGUID;                      // was invited to certain BG
     uint32  ArenaTeamRating;                                // if rated match, inited to the rating of the team
     uint32  ArenaMatchmakerRating;                          // if rated match, inited to the rating of the team
+    uint32  OpponentsTeamId;                                // for compete again
```

Comment:
comment field grammar suggestion:
```cpp
    uint32  OpponentsTeamId;                                // for instant rematch with the same team
```

Reply:
(in case I have misunderstood the purpose of this function, please explain)

Reply:
Gets the value of the previous opponent to exclude it from the queue

Reply:
OK, so that is to store the team ID for a new match, right?
Maybe `// rematch with the current team`
(`OpponentsTeamId` itself explains the rest)

Reply:
It is not the current one, it is from the previous opponent. It will be excluded from the queue until the timer is met and they can compete again.

Reply:
Could be a useful renaming, makes more sense in this case.
`// team previously excluded from the queue`

Reply:
hmm... It is excluded for the current queue